File: tests/fixtures/java_maven_sample/src/main/java/com/example/myapp/main/Application.java
package com.example.myapp.main;

import com.example.myapp.domain.User;
import com.example.myapp.usecase.UserService;
import com.example.myapp.infrastructure.UserRepo;

/**
 * Application is the entry point — imports from all layers.
 */
public class Application {
    public static void main(String[] args) {
        UserRepo repo = new UserRepo();
        UserService service = new UserService();
        User user = service.createUser(1, "Alice");
        System.out.println(user.getName());
    }
}

File: tests/fixtures/kotlin_gradle_sample/src/main/kotlin/com/example/myapp/main/Application.kt
package com.example.kotlinapp.main

import com.example.kotlinapp.domain.User
import com.example.kotlinapp.usecase.UserService
import com.example.kotlinapp.infrastructure.UserRepo

fun main() {
    val service = UserService()
    val repo = UserRepo()
    println(service.createUser(1, "Alice"))
}
